packages/x-date-pickers/src/internals/models/props/basePickerProps.tsx

```diff
@@ -77,4 +77,10 @@ export interface BaseNonRangeNonStaticPickerProps {
    * Pass a ref to the `input` element.
    */
   inputRef?: React.Ref<HTMLInputElement>;
+  /**
+   * The `name` attribute used on the `field` component.
```

Comment:
We are not super consistent in how we phrase our JSDoc.
But the following is more in line with the majority of our props:

```suggestion
   * Name attribute used by the `input` element in the Field.
```

The same modification can be applied on the pro package.